static/docs/commands-reference/status.md

```diff
@@ -53,17 +54,30 @@ be rerun if `dvc repro` were executed.
 If instead, differences are detected, `dvc status` lists those changes. For each
 DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
 differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+shown, and additionally a status word is shown describing the changes in
+checksum or status for both local and cloud workspace:
 
 - For the local workspace:
-  - _changed_ means the named file has changed
+
+  - _changed checksum_ means actual checksum of the DVC-file doesn't match the
```

Comment:
can you show an example how does it look like? The same for always changed and changed checksum. Let's artificially create a few example and see how does it look like, please.

there is still some disconnect in this list, in what is described in the paragraph above.

Specifically,  _changed_, _changed checksum_, and _ always changed_ are statuses of the DVC-file itself.

Reply:
@shcheklein shouldn't we add some examples in simple usage section?

Reply:
that would be ideal, sure!

Reply:
@shcheklein I have added the examples in the simple usage section kindly check them out.

Reply:
this is cool. So, you see:

```
test.dvc:
	changed outs:
		modified:           test
	changed checksum
	always changed
```

Items like `modified` are actually not on the same level of hierarchy as `always changed`.

Why don't we come with a better description that explains these two lists separately. And explains the connection between them.